src/ml-search.service.js

```diff
@@ -1137,16 +1141,16 @@
       }
 
       params = params || {};
-      params.start = params.start || 1;
-      params.limit = params.limit || 20;
+      params.start = params.start !== undefined ? params.start : 1;
+      params.limit = params.limit !== undefined ? params.limit : 20;
```

Comment:
This more explicit check is to allow falsy values as parameters (such as `limit=0`)?